src/build/env.py

```diff
@@ -38,6 +42,24 @@ def _restore_env(self):  # type: () -> None
     def _get_env_path(self, path):  # type: (str) -> Optional[str]
         return sysconfig.get_path(path, vars=self._env_vars)
 
+    def _symlink_relative(self, path):  # type: (Optional[str]) -> None
+        if not path:  # pragma: no cover
+            return
+
+        prefix = sysconfig.get_config_var('prefix')
+        if prefix and path and path.startswith(prefix):
+            new_path = os.path.join(self._path, path[len(prefix + os.pathsep):])
+            if not os.path.exists(new_path):
+                try:
+                    os.makedirs(os.path.dirname(new_path))
```

Comment:
Why not just use exists_ok flag here?

Reply:
Could you need here ``parents=True``?

Reply:
I think `exists_ok=True` was added in Python 3.2. As long as this package still supports 2.7, we can't use it (though maybe it could be moved into a compatibility function) to make it easier to replace later. I think `parents` is not a valid argument to this function (it always creates the parent directories, which is the difference between `makedirs` and `mkdir`.

Reply:
Correct, Python 2.7 does not have `exists_ok`.